src/passes/identifierManglerPass/declarationNameMangler.ts

```diff
@@ -85,17 +100,22 @@ export class DeclarationNameMangler extends ASTMapper {
       : fd.name;
   }
 
+  // Return a new id formatted to achieve the minimum lenght
+  getFormattedId(): string {
+    return (this.lastUsedId++).toString().padStart(this.initialIdWidth, '0');
+  }
+
   // This strategy should allow checked demangling post transpilation for a more readable result
   createNewInternalFunctionName(existingName: string): string {
-    return `${MANGLED_INTERNAL_USER_FUNCTION}${this.lastUsedFunctionId++}_${existingName}`;
+    return `${MANGLED_INTERNAL_USER_FUNCTION}_${this.getFormattedId()}_${existingName}`;
   }
 
   createNewTypeName(existingName: string): string {
-    return `${MANGLED_TYPE_NAME}${this.lastUsedTypeId++}_${existingName}`;
+    return `${MANGLED_TYPE_NAME}_${this.getFormattedId()}_${existingName}`;
   }
 
   createNewVariableName(existingName: string): string {
-    return `${MANGLED_LOCAL_VAR}${this.lastUsedVariableId++}_${existingName}`;
+    return `${MANGLED_LOCAL_VAR}_${this.getFormattedId()}_${existingName}`;
```

Comment:
Does the extra '_' change to naming should go here or in the constants (i.e `MANGLED_LOCAL_VARS` ...)?. Also, I think it would be wise if these name mangling constants could be used as well in `CairoStubProcessor` pass regex expressions to have a single source of truth and avoid future headaches.

Reply:
That's a good idea.